src/PSM.sol

```diff
@@ -65,7 +65,8 @@ contract PSM {
         address assetOut,
         uint256 amountIn,
         uint256 minAmountOut,
-        address receiver
+        address receiver,
+        uint16  referralCode
```

Comment:
Let's extend this to `uint256`. I'm not sure why Aave made it smaller (does it save gas..?), but we don't have to blindly duplicate them here.

Reply:
Updated